MetafileImporter/qSlicerMetafileReader.cxx

```diff
@@ -86,7 +90,7 @@ qSlicerIO::IOFileType qSlicerMetafileReader::fileType() const
 //-----------------------------------------------------------------------------
 QStringList qSlicerMetafileReader::extensions() const
 {
-  return QStringList() << "Sequence Metafile (*.seq.mha *.seq.mhd *.mha *.mhd)";
+  return QStringList() << "Sequence Metafile (*.seq.mha *.seq.mhd *.mha *.mhd *.igsio.mha *.igsio.nrrd)";
```

Comment:
.igsio.mha and .igsio.nrrd extensions may be a bit long and I feel including IO for a file is somewhat redundant. Maybe .igs.mha and .igs.nrrd extensions would be better?

@adamrankin @ungi What do you think? What composite extension would you prefer for tracked image sequences? .seq.nrrd and .seq.mha would not work in general, because this file extensions are used for simple (non-tracked) volume sequences.

Reply:
I don't have much of a preference. I agree that igsio is quite long.
I've implemented the `*.igs.*` extension.